specification/_json_spec/indices.explain_data_lifecycle.json

```diff
@@ -1,10 +1,10 @@
 {
   "indices.explain_data_lifecycle": {
     "documentation": {
-      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/dlm-explain-lifecycle.html",
-      "description": "Retrieves information about the index's current DLM lifecycle, such as any potential encountered error, time since creation etc."
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams-explain-lifecycle.html",
```

Comment:
Just noting for other reviewers: some of these links to `current` docs are 404-ing for now, but I checked and they do all exist on the `master` docs.

Reply:
Thank you @JoshMock for this remark. You are right, recently we decided to hide the preview documentation for the data stream lifecycle because it was getting our users confused. Docs will go live along with the feature.